indico/modules/receipts/client/js/templates/TemplateListPane.jsx

```diff
@@ -183,16 +184,33 @@ export default function TemplateListPane({
           <h3>
             <Translate>Custom templates</Translate>
           </h3>
-          <Link to={addTemplatePageURL(targetLocator)}>
-            <Button
-              primary
-              size="mini"
-              title={Translate.string('Add a new blank receipt/certificate template')}
-            >
-              <Icon name="plus" />
-              <Translate>Add new</Translate>
-            </Button>
-          </Link>
+          <Dropdown
+            button
+            labeled
+            title={Translate.string('Add a new template')}
+            icon="plus"
+            text={Translate.string('Add new')}
+            className="mini primary icon"
+            direction="left"
+          >
+            <Dropdown.Menu>
+              <Dropdown.Item
+                as={Link}
+                to={addDefaultTemplateURL({...targetLocator, template_name: 'receipt'})}
+                text={Translate.string('Receipt template')}
+              />
+              <Dropdown.Item
+                as={Link}
+                to={addDefaultTemplateURL({...targetLocator, template_name: 'attendance'})}
+                text={Translate.string('Certificate of Attendance template')}
+              />
```

Comment:
Can't we do this dynamically instead of hardcoding a list? It's probably just a matter of time anyway before someone adds a plugin signal so a plugin can provide more default templates...